test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/SignCommandTests.cs

```diff
@@ -99,7 +99,7 @@ public void SignCommand_SignPackageWithInvalidEkuFails()
                 result.Success.Should().BeFalse();
                 result.AllOutput.Should().Contain(_noTimestamperWarningCode);
                 result.AllOutput.Should().Contain(_invalidEkuErrorCode);
-                result.AllOutput.Should().Contain("NotValidForUsage");
+                result.AllOutput.Should().Contain("The certificate is not valid for the requested usage");
```

Comment:
In reference to [this comment](https://github.com/NuGet/NuGet.Client/pull/1896/files#diff-d216918a0fbcc8eca7eb5a4126c14105R161), are tests ever run on machines with non-English .NET Framework/Core?

`X509ChainStatusFlags.NotValidForUsage.ToString()` would be language agnostic.

Reply:
Because this string is localized and not generated by NuGet. Should I remove the assertion of the message and just assert the nuget error code?

Reply:
Our CI is always run in machines in English culture.